src/pages/portfolio/Fees.tsx

```diff
@@ -46,35 +46,34 @@ export const Fees = () => {
   }, [userStats]);
 
   const AdditionalConditions = useCallback(
-    ({
-      totalShare,
-      makerShare,
-      isAdditional,
-    }: {
+    (conditions: {
```

Comment:
nit: is this an eslint rule where they prefer destructuring within the cb?

Reply:
Nope this is an eslint bug/false positive - it thinks this function is a react component and is angry about unused props for some reason. Maybe bumping eslint would fix it.

Reply:
Well this is correctly identifying this function as a react functional component but I don't know why it thinks the props are unused.

Reply:
Seems this is a known issue too and latest eslint says it ignores these, not sure why it's not working here.